src/main/java/no/nav/familie/integrasjoner/personopplysning/PersonopplysningerController.kt

```diff
@@ -78,4 +89,13 @@ class PersonopplysningerController(private val personopplysningerService: Person
                                   defaultValue = "false") medHistorikk: Boolean): Ressurs<FinnPersonidenterResponse> {
         return success(personopplysningerService.hentIdenter(ident.ident, tema, medHistorikk))
     }
+
+
+    @PostMapping(produces = [MediaType.APPLICATION_JSON_VALUE], path = ["strengeste-adressebeskyttelse/"])
```

Comment:
Litt usikker på om pathen spegler helt att det gjelder adressebeskyttelsen på personen elle alle relasjoner. 
Hva skal denne brukes til? `hentPersonMedRelasjoner` henter alle relasjoner, dvs andre identer i sivilstand, fullmakt etc, er det ønskelig? 
```suggestion
    @PostMapping("strengeste-adressebeskyttelse")
```

Reply:
Ja enig i den. Skal vi endre til: 
strengeste-adressebeskyttelse-for-person-med-relsjoner (blir jo litt lang 😬 )

Jepp - det er ønskelig. Vi skal i første omgang bruke den i ef-sak -> brev. Dersom gradering = STRENG på person, eller relasjon skal vi anonymisere signaturer i brev.

Reply:
Ok. `strengeste-adressebeskyttelse-relasjoner` kanskje?

Reply:
Bra forslag!